06_gpu_and_ml/stable_diffusion_cli.py

```diff
@@ -115,6 +115,8 @@ def download_models():
 
 class StableDiffusion:
     def __enter__(self):
+        os.environ["SAFETENSORS_FAST_GPU"] = "1"
```

Comment:
does this matter? wasn't clear to me

Reply:
I just measured. It makes no difference. Also looking at how this is implemented in `diffusers`, it also doesn't seem to do anything. Just removed.